specs/experimental/fault-proof/stage-one/fault-dispute-game.md

```diff
@@ -459,3 +479,12 @@ Each move bisects the historical state of L2 and eventually, `MAX_GAME_DEPTH` is
 can be settled conclusively. Dishonest players are disincentivized to participate, via backwards induction,
 as an invalid claim won't remain uncontested. Further incentives can be added to the game by requiring
 claims to be bonded, while rewarding game winners using the bonds of dishonest claims.
+
+### Finalization
+
+Once the game is resolved, if the claim is shown to be valid, the FDG reports its state to the
+Anchor State Registry. The Anchor State Registry verifies that the request to update an anchor
+state comes from a FDG contract created by the Dispute Game Factory contract, confirms that the
+game resolved in favor of the defender, and confirms that the updated state would be newer than
```

Comment:
> would be newer than

How exactly are two states ordered?

Reply:
We order ORs by L2 block number. With the registry being generic over game type, we may need custom handlers for each game type?

This could maybe done by having an opaque `bytes` in storage for each game type that holds the calldata payload for the dispute game contracts, to fetch the data necessary to correctly order things. i.e., for FDGs, we'd just store `abi.encode("l2BlockNumber()")`.

Reply:
I vote that we make this a FaultDisputeGame-specific construct. It doesn't need to be generic for all dispute games.